app/oauth2/access-token-request.js

```diff
@@ -28,7 +28,7 @@ function accessTokenRequest(request) {
     .then(response => response.status === 200 ? response : response.text().then(text => Promise.reject(new Error(text))))
     .then(response => response.json())
     .catch(error => {
-      console.error('Failed to obtain access token:', error);
+      // console.error('Failed to obtain access token:', error);
```

Comment:
Instead of losing the error logging altogether, can you replace with something like:
```
const logger = Logger.getLogger(__filename);
```
before the `return fetch(...)`, then
```
logger.error("Failed to obtain access token:", error);
```
in the `catch` block?

See https://github.com/hmcts/ccd-admin-web/blob/master/src/main/oauth2/access-token-request.ts for how it's been done there.

Reply:
DONE